pkg/controllers/management/node/utils.go

```diff
@@ -227,7 +227,7 @@ func (m *Lifecycle) reportStatus(stdoutReader io.Reader, stderrReader io.Reader,
 			// under trace and don't add to the v3.NodeConditionProvisioned.Message
 			logrus.Tracef("[node-controller] %v", msg)
 		} else {
-			logrus.Infof("[node-controller] %v", msg)
+			logrus.Infof("[node-controller] [cluster-id: %s] %v", node.ObjClusterName(), m.clusterManager., msg)
```

Comment:
I think you might have left `m.clusterManager.` in accidentally

Reply:
Indeed. I forgot to add the fix.